packages/tocco-ui/src/utilStyles/doesReuseAllValuesAsKeys.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Enforce that all values of a flat object are used as keys in another object.
+ * @param  {object} completeMap            A flat object with contains a full list of
+ *                                         all allowed keys defined as values.
+ * @param  {object} potentialIncompleteMap A flat object which is does not reuse all values as keys.
+ * @return {boolean}                       Return true if all values are reused as keys.
+ */
+
+const doesReuseAllValuesAsKeys = (completeMap, potentialIncompleteMap) => {
```

Comment:
Is there a better name for this function? Maybe `assertObjectKeysMatch` or something like that?